sci-geosciences/osgearth/osgearth-2.8.ebuild

```diff
@@ -0,0 +1,74 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit cmake-utils
+
+DESCRIPTION="Dynamic map generation toolkit for OpenSceneGraph"
+HOMEPAGE="http://osgearth.org/"
+SRC_URI="https://github.com/gwaldron/osgearth/archive/${P}.tar.gz"
+
+LICENSE="LGPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc qt5"
+
+RDEPEND="
+	dev-db/sqlite:3
+	>=dev-games/openscenegraph-3.4[curl,qt5?]
+	dev-libs/protobuf
+	dev-libs/tinyxml
+	net-misc/curl
+	sci-libs/gdal
+	sci-libs/geos
+	sys-libs/zlib[minizip]
+	virtual/opengl
+	x11-libs/libX11
+	qt5? (
+		dev-qt/qtcore:5
+		dev-qt/qtgui:5
+		dev-qt/qtopengl:5
+	)"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig
+	doc? ( dev-python/sphinx )"
+
+S=${WORKDIR}/${PN}-${P}
+
+PATCHES=(
+	#"${FILESDIR}"/${PN}-2.6-cmake-options.patch
+	#"${FILESDIR}"/${PN}-2.7-linker.patch
```

Comment:
Have you verified that the PATCHES aren't needed anymore? Or is it just that the patches don't apply cleanly anymore? If you're sure the fixes have been applied upstream, you can remove the `PATCHES` thing completely, otherwise you need to port the patches.